sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -557,7 +557,10 @@ async def _read(self, n, buffer=None, **kwargs):  # pylint: disable=unused-argum
                     n = 0
             return view
         except asyncio.TimeoutError as te:
-            raise ConnectionError('Receive timed out (%s)' % te)
+            raise ConnectionError('recv timed out (%s)' % te)
```

Comment:
curious why we're changing the message here?

Reply:
(Anna mentioned in one of my PRs that we want to capitalize the first letter of the error message, so I changed this in on eof my PRs)